blockcypher/api.py

```diff
@@ -1499,6 +1500,15 @@ def create_unsigned_tx(inputs, outputs, change_address=None,
             inputs_cleaned.append({
                 'addresses': [address, ],
                 })
+        elif 'prev_hash' and  'output_index' in input_obj:
+            prev_hash = input_obj['prev_hash']
```

Comment:
It was meant for line 1509
(Apologies, git and coding are never as easy using mobile devices)